app/Jobs/ProvisionQueryserviceNamespaceJob.php

```diff
@@ -12,12 +15,15 @@ class ProvisionQueryserviceNamespaceJob extends Job
 {
     private $namespace;
     private $maxFree;
+    private $request;
 
     /**
      * @return void
      */
-    public function __construct($namespace = null, $maxFree = null)
+    public function __construct($namespace = null, $maxFree = null, HttpRequest $request = null)
```

Comment:
HttpRequest cant be injected in constructor

Reply:
done